doc/build-windows.md

```diff
@@ -66,12 +69,29 @@ See also: [dependencies.md](dependencies.md).
 
 ## Building for 64-bit Windows
 
-To build executables for Windows 64-bit, install the following dependencies:
+The first step is to install the mingw-w64 cross-compilation tool chain. Due to different Ubuntu
+packages for each distribution and problems with the Xenial packages the steps for each are different.
+
+Ubuntu Trusty 14.04:
 
     sudo apt-get install g++-mingw-w64-x86-64 mingw-w64-x86-64-dev
 
-Then build using:
+Ubuntu Xenial 16.04 and Windows Subsystem for Linux ([^1][^2]):
```

Comment:
Might want to remove WSL here, since it will no longer be true when the user runs a different ubuntu version than xenial in their WSL.

Reply:
Wouldn't it be better to leave it there and update the doc when WSL gets updated? I suspect the majority of devs reading this doc will be looking for WSL instructions.